gui/src/components/Taipy/DateSelector.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import { render, fireEvent } from "@testing-library/react";
+import { render } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import "@testing-library/jest-dom";
 
-import { LocalizationProvider } from "@mui/lab";
+import { LocalizationProvider } from "@mui/x-date-pickers";
```

Comment:
Hmmm... Introduction of Mui X...

Reply:
it replaces @mui/lab for date pickers and I'd say it is more stable than lab